site/src/documents/pages/blocks/FooterContentBlock.tsx

```diff
@@ -0,0 +1,138 @@
+import { PropsWithData, withPreview } from "@comet/cms-site";
+import { FooterContentBlockData } from "@src/blocks.generated";
+import { DamImageBlock } from "@src/common/blocks/DamImageBlock";
+import { LinkBlock } from "@src/common/blocks/LinkBlock";
+import { RichTextBlock } from "@src/common/blocks/RichTextBlock";
+import { Typography } from "@src/common/components/Typography";
+import * as React from "react";
+import styled from "styled-components";
+
+export const FooterContentBlock = withPreview(
+    ({ data: { text, image, linkList, copyrightNotice } }: PropsWithData<FooterContentBlockData>) => {
+        return (
+            <Root>
+                <ImageTextWrapper>
+                    <ImageWrapper>
+                        <DamImageBlock data={image} aspectRatio="1/1" objectFit="contain" />
+                    </ImageWrapper>
+                    <RichTextWrapper>
+                        <RichTextBlock data={text} />
+                    </RichTextWrapper>
+                </ImageTextWrapper>
+                <HorizontalLine />
+                <LinkCopyrightWrapper>
+                    {!!linkList.blocks.length && (
+                        <LinksBlock>
+                            {linkList.blocks.map((block) => (
+                                <LinkText as={LinkBlock} key={block.key} data={block.props.link} variant="p200">
+                                    {block.props.text}
+                                </LinkText>
+                            ))}
+                        </LinksBlock>
+                    )}
+                    {copyrightNotice && <CopyrightNotice variant="p200">{copyrightNotice}</CopyrightNotice>}
+                </LinkCopyrightWrapper>
+            </Root>
+        );
+    },
+    { label: "Footer" },
+);
+
+const Root = styled.footer`
+    margin-top: auto;
+    background-color: ${({ theme }) => theme.palette.grey["900"]};
+    color: ${({ theme }) => theme.palette.grey["50"]};
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 60px 16px;
```

Comment:
```suggestion
    padding: 60px ${({ theme }) => theme.spacing.D100};
```

maybe you can use a value from the spacings here?

Reply:
https://github.com/vivid-planet/comet-starter/pull/282/commits/19ddced1ce8a8eb24c6c8d21c915b6952149b36d